tests/tox_env/test_tox_env_api.py

```diff
@@ -80,6 +80,26 @@ def test_tox_env_pass_env_literal_miss() -> None:
     assert not env
 
 
+def test_tox_env_pass_env_fail(tox_project: ToxProjectCreator) -> None:
+    prj = tox_project(
+        {
+            "tox.ini": """[testenv]
+    passenv = MYENV YOURENV, THEIRENV
+    commands=python -c 'import os; print("MYENV", os.getenv("MYENV"))'""",
+        },
+    )
+
+    result = prj.run("r")
+
+    result.assert_failed(1)
+    out = (
+        r"py: failed with pass_env/passenv variable can't have values "
+        r"containing blanks like \['MYENV YOURENV'\]; "
+        r"a comma is possibly missing.*"
+    )
```

Comment:
Let's not use regex assert, let's do a literal match, please.  So we print just the error message and notthing more.